test/javascripts/acceptance/topic-calendar-holidays-test.js

```diff
@@ -0,0 +1,35 @@
+import { visit } from "@ember/test-helpers";
+import { test } from "qunit";
+import { acceptance, fakeTime } from "discourse/tests/helpers/qunit-helpers";
+import eventTopicFixture from "../helpers/event-topic-fixture";
+
+acceptance("Discourse Calendar - Topic Calendar Holidays", function (needs) {
+  needs.hooks.beforeEach(function () {
+    this.clock = fakeTime("2023-12-10T00:00:00", "America/Los_Angeles", true);
+  });
+
+  needs.hooks.afterEach(function () {
+    this.clock.restore();
+  });
+
+  needs.settings({
+    calendar_enabled: true,
+  });
+
+  needs.pretender((server, helper) => {
+    server.get("/t/252.json", () => {
+      return helper.response(eventTopicFixture);
+    });
+  });
+
+  test("renders calendar holidays with fullDay='true'", async (assert) => {
+    await visit("/t/-/252");
+
+    assert
+      .dom(".fc-week:nth-child(5) .fc-content-skeleton tbody td:first-child")
+      .hasClass(
+        "fc-event-container",
+        "Italian Christmas Day is displayed on Monday 2023-12-25"
```

Comment:
Interesting. I was wondering how "Italian" was decided, then saw the fixture below. Would it be more obvious if you make the fixture additions in line 21 of this file instead? (i.e. import then add to the array within this test file)